ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/workspace-item/workspace-item.tsx

```diff
@@ -67,6 +67,12 @@ const Icon = styled.span`
   margin-right: calc(${props => props.theme.minimumSpacing} / 2);
 `
 
+const Author = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
❓ Does this one need `display: block` too or can we remove that in the other `Author`?